Dockerfile

```diff
@@ -13,17 +13,22 @@ RUN yarn production
 FROM python:3.8-slim
 
 ENV DJANGO_SETTINGS_MODULE=zoo.base.settings
+ENV POETRY_VERSION=1.1.4
```

Comment:
do we want to pin version?is there any reason why?

Reply:
I’d like to know why not pinning versions.

Reply:
checked it up and indeed in docker files its better to pin...